forc-plugins/forc-doc/src/render/item/context.rs

```diff
@@ -350,6 +350,42 @@ impl ItemContext {
                 }
             }
         }
+
+        if let Some(inherent_impls) = &self.inherent_impls {
+            let mut doc_links = Vec::new();
+            for inherent_impl in inherent_impls {
+                for item in inherent_impl.impl_trait.items.clone() {
```

Comment:
we can avoid a clone here
```suggestion
                for item in &inherent_impl.impl_trait.items {
```

Reply:
Resolving this as this is addressed in a2b1e97